civis_jupyter_notebooks/platform_persistence.py

```diff
@@ -24,7 +24,7 @@ def initialize_notebook_from_platform(notebook_path):
     notebook_model = client.notebooks.get(os.environ['PLATFORM_OBJECT_ID'])
 
     logger.info('Pulling contents of notebook file from S3')
-    r = requests.get(notebook_model.notebook_url)
```

Comment:
do we need any retry logic? Or is this retaining existing behavior, just more clearly, if platform times out after 60 seconds anyway?

Reply:
It's retaining the existing behavior but it's more explicit.  I think requests won't timeout _ever_ if the server doesn't timeout, so `bandit` throws an error about requests without timeout: https://bandit.readthedocs.io/en/1.7.5/plugins/b113_request_without_timeout.html

Since platform will timeout after 60s, this is mainly just for appeasing the bandit tool, and it's good practice.